### DISCUSSION TITLE
Duvidas com como PCB

### DISCUSSION THREAD

**Discussion by romehi** [sanni/cartreader#157] [General Chat]
Bom dia!
Sanni tudo bem?

Sou iniciante nessa area  e seu projeto esta sendo o eu primeiro.
Gostaria de agradecer pois por causa dele estou entrando na area.
Estou com duvidas referente as PCB dos cartuchos de Snes, MD e 64, quais são recomendados para usar no projeto, e se tem algum arquivo onde consigo solicitar a fabricação desses PCB de cartuchos, ou a compra do projeto gerber.

**Comment by sanni** [comment]
Hi,

the current functions for flashing repros from Aliexpress for SNES/MD/N64 are just examples. You have to edit them to match the PCBs you got. Which is somewhat hard to do though since you will have to read the datasheets of the flashrom on the PCB you bought. 
Sadly there is no way telling what PCBs you get when ordering from Aliexpress since the vendors use new types all the time.

**Reply by romehi** [reply]
Entendi, muito obrigado pelo esclarecimento.
Você me recomenta alguma loja do Aliexpress para a compra da pcb?

**Comment by fiatbravodriver** [comment]
Hi, 

Have a look at my post. 

https://github.com/sanni/cartreader/discussions/155

Maybe you could take a photo of the Flash Memory and see if the chips have any ID and maybe we could get an update done with as many as we know of to save Sanni work?

I wish I had the knowledge to be able to do this myself.
